MAVProxy/mavproxy.py

```diff
@@ -989,6 +989,10 @@ def set_mav_version(mav10, mav20, autoProtocol, mavversionArg):
           print("ERROR: mavproxy takes no position arguments; got (%s)" % str(args))
           sys.exit(1)
 
+    if opts.moddebug not in [0, 1, 2, 3]:
```

Comment:
This won't catch someone doing a `set moddebug 5`

Reply:
The `set moddebug X` is already covered by existing code. The issue was the exisitng code did not check the commandline parameters.